cleanup_utils/playbooks/requirements.txt

```diff
@@ -0,0 +1 @@
+ansible==2.1.2.0
```

Comment:
Any reason not to use the latest ansible?

Reply:
@UmanShahzad, I just used the same version currently used by the load balancer setup playbook. I can update the version to the latest one.